src/tree/ContainerAppItem.ts

```diff
@@ -60,23 +59,22 @@ export class ContainerAppItem implements ContainerAppsItem {
     portalUrl: Uri = createPortalUrl(this.subscription, this.containerApp.id);
 
     async getChildren(): Promise<TreeElementBase[]> {
+        // this should have configuration and revisions
```

Comment:
Was this like a todo comment?  I see the configurations and revisions below already.

Reply:
Yeah haha, notes while I am thinking of the UI. Whoops 😳